lib/ReadOnlyMiddleware.ts

```diff
@@ -0,0 +1,11 @@
+import { Request, Response, NextFunction } from 'express';
+
+const MONGOKU_READ_ONLY_MODE = process.env.MONGOKU_READ_ONLY_MODE === 'true';
+
+export function readOnlyMode(_: Request, res: Response, next: NextFunction) {
+    if (MONGOKU_READ_ONLY_MODE) {
+        res.sendStatus(401);
```

Comment:
```suggestion
        res.status(401).json();
```
weird behavior otherwise when trying to edit a file in my tests